docs/src/main/sphinx/release/release-466.md

```diff
@@ -0,0 +1,80 @@
+# Release 466 (27 Nov 2024)
+
+## General
+
+* Improve performance of queries with grouping on joins. ({issue}`23812`)
+* Improve client protocol throughput by introducing the spooling protocol.
+  ({issue}`24214`)
+* Improve performance of querying `system.jdbc.tables` for multiple catalogs.
```

Comment:
Need clarification here @piotrrzysko @raunaqmorarka @Praveen2112 @hashhar .. what is the user facing impact here .. when and where is the system.jdbc.tables info accessed?

Reply:
Replied on the original PR already.

People can query it on their own to see metadata. BI tools for sure use it to inspect metadata.

Reply:
Great.. updated it now.